### DISCUSSION TITLE
New Apple emojis osx 15.4

### DISCUSSION THREAD

**Discussion by Dsinghbailey** [missive/emoji-mart#760] [Q&A] [answered]
Hello! First of all I'm a big fan. We're using emoji-mart in our chat platform.

We've had a few users request emojis in apple's 15.4 release (specifically the salute). I can add them as custom emojis, but I figured I'd check here first.

Any plans to add the new emojis from ios 15.4?

**Comment by EtienneLem** [comment, marked as answer]
The [🫡 Saluting Face](https://emojipedia.org/saluting-face) emoji is part of [Emoji 14.0](https://emojipedia.org/emoji-14.0) (available on iOS 15.4). Emoji 14 is supported in EmojiMart v5+. You can check https://missiveapp.com/open/emoji-mart to see all the available emojis.
